server/src/main/java/org/opensearch/threadpool/ThreadPoolStats.java

```diff
@@ -197,6 +215,7 @@ static final class Fields {
         static final String REJECTED = "rejected";
         static final String LARGEST = "largest";
         static final String COMPLETED = "completed";
+        static final String WAIT_TIME = "total_wait_time";
```

Comment:
@jed326 my apologies, I missed that previously, but all our time-based stats have time unit suffix (so no need to guess what it is, fe OsStats, ProfileResult, .... ):

```suggestion
        static final String WAIT_TIME = "total_wait_time_nanos";
```

Reply:
Since we're using the `TimeValue` class I think the units is handled by the class itself -- see the examples I pasted in the PR description.

Reply:
https://github.com/opensearch-project/OpenSearch/blob/56a87362c728f4d6c13f74eaacea8c7e50ed6a83/server/src/main/java/org/opensearch/index/search/stats/SearchStats.java#L371-L376

So looks like for other time values we use this `humanReadableField` which 

https://github.com/opensearch-project/OpenSearch/blob/78eea275a3e1812a008e815d846ca871d7f09d20/libs/core/src/main/java/org/opensearch/core/xcontent/XContentBuilder.java#L961-L983

And that's controlled by the `?human` parameter. So yeah, to keep with the same format let me change this metric to use the same thing.